File: CPP/Baju.cpp
#include <iostream>
#include <string>
using namespace std;
#include "Aksesoris.cpp"

class Baju : public Aksesoris
{
private:
    int size;
    string merk;

public:
    Baju() {}

    Baju(int size, string merk)
    {
        this->size = size;
        this->merk = merk;
    }
    int get_size()
    {
        return this->size;
    }
    void set_size(int size)
    {
        this->size = size;
    }
    string get_merk()
    {
        return this->merk;
    }
    void set_merk(string merk)
    {
        this->merk = merk;
    }

    ~Baju() {}


};

File: PHP/Baju.php
<?php

require_once 'Aksesoris.php';

class Baju extends Aksesoris {
    private $size;
    private $merk;

    // Konstruktor default
    public function __construct($id = 0, $nama_produk = "", $harga_produk = 0, $stok_produk = 0, $jenis_aksesoris = "", $bahan_aksesoris = "", $warna_aksesoris = "", $size = 0, $merk = "") {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $jenis_aksesoris, $bahan_aksesoris, $warna_aksesoris);
        $this->size = $size;
        $this->merk = $merk;
    }

    public function getSize() {
        return $this->size;
    }

    public function setSize($size) {
        $this->size = $size;
    }

    public function getMerk() {
        return $this->merk;
    }

    public function setMerk($merk) {
        $this->merk = $merk;
    }
}

?>
